components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div>
+        <span class="mb-3.5 text-center text-primary-text text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+            {{ $t('modSubmissionForm.healthcareProfessionalId') }}
+        </span>
+        <div class="flex flex-col mt-4">
+            <div class="justify-start items-start flex">
+                <input 
+                    type='text'
+                    :placeholder=placeholderText
+                    class="mb-5 px-3 py-3.5 w-96 h-12 bg-white rounded-lg border border-primary-text-muted text-primary-text text-sm font-normal font-['Noto Sans JP'] placeholder-primary-text-muted"
+                />
+                <SVGLookingGlass role="img" title="searching icon" class="relative right-8 top-3 w-6 h-6"/>
+            </div>
+        </div>
+    </div> 
+</template>
+
+<script lang="ts" setup>
+import {ref, Ref} from 'vue'
+
+const placeholderText: Ref<string> = ref('Search by name or ID to add a doctor')
```

Comment:
this should still be pulled from i18n. I think don't think it need to be a ref vs just a variable?

Reply:
Ah sorry I totally misunderstood what you and LaShawn meant, but now I understand thank you! Will fix this 🙏